sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -1445,6 +1440,40 @@ SIREPO.app.directive('simplePanel', function(appState, panelState) {
     };
 });
 
+SIREPO.app.directive('simulationStoppedStatus', function(authState) {
+    return {
+        restrict: 'A',
+        scope: {
+            simState: '=simulationStoppedStatus',
+        },
+        template: [
+            '<div class="col-sm-12" ng-bind-html="message()"><br><br></div>',
+        ].join(''),
+        controller: function($scope, $sce) {
+            $scope.message = function() {
+                if ($scope.simState.isStatePurged()) {
+                    return $sce.trustAsHtml([
+                        '<div>Simulation data purged on ' + $scope.simState.getDbUpdateTime() + ' UTC.</div>',
```

Comment:
`getDbUpdateTime` should return a time which is rendered in the local time zone. There's no need to use UTC for end-users.

Reply:
https://github.com/radiasoft/sirepo/commit/baaf84a3bdfc7d0bba4ec6793c0eb670645b610f